apispec/ext/flask.py

```diff
@@ -72,6 +111,20 @@ def path_from_view(spec, view, **kwargs):
     path = Path(path=path, operations=operations)
     return path
 
+def path_from_method_view(spec, method_view, **kwargs):
```

Comment:
Using 'method_view' (not 'view' as used in existing helper function) forces user to distinguish the view is derived from MethodView or  not.